gitgud/__main__.py

```diff
@@ -197,11 +197,11 @@ def handle_levels(self, args):
 
         print("Currently on level: \"{}\"\n".format(cur_level.name))
         
-        for level in all_levels.values():
+        for level in all_levels:
             # TODO Add description
             # 10 characters for the short IDs. 
             print("Level {:<10} :{:>2} challenge{}".format("\"" + level.name + "\"", len(level.challenges), ("", "s")[len(level.challenges) > 1]))
-            for index, challenge in enumerate(level.challenges.values()):
+            for index, challenge in enumerate(level.challenges):
```

Comment:
Once we update, we should be able to do `for index, challenge in enumerate(level):`